Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN CGO_ENABLED=0 GOOS=linux GO11MODULE=on go build -a -o /main .
 FROM gcr.io/distroless/static:nonroot
 COPY --from=builder --chown=nonroot:nonroot /main /kubernetes-event-exporter
 
-USER nonroot
+# https://github.com/GoogleContainerTools/distroless/blob/main/base/base.bzl#L8C1-L9C1
+USER 65532
```

Comment:
```
Error: container has runAsNonRoot and image has non-numeric user (nonroot), cannot verify user is non-root (pod: "event-exporter-6d778fb78d-4wttt_monitoring(fed8a68a-8175-42c9-a542-2f3f6d56c385)", container: event-exporter)
```

To allow Kubernetes to determine that a container user is non-root, it must be specified by a UID.